### DISCUSSION TITLE
Ability to hide 'Exemplars' checkbox in Prometheus query editor

### DISCUSSION THREAD

**Discussion by sd2k** [grafana/grafana#43658] [General]
The Prometheus query editor has a checkbox to indicate whether it should include a query for exemplars, which is [hidden in the special case where the query is run from unified alerting](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/components/PromQueryEditor.tsx#L116). We also need to hide this checkbox for Grafana ML, so it'd be great if there was a way to specify this option when creating the component somehow.

Happy to take a stab at this if anyone can point me in the right direction. My first thoughts were that `QueryEditorProps` doesn't allow for customisation of the component itself so I'm not sure what the best way to approach this is.

**Comment by ivanahuckova** [comment]
@sd2k I did this in this PR https://github.com/grafana/grafana/pull/41607. I think you could do similar/same thing for ML use case. Let me know if it is clear and if I can help in any way. :)

**Reply by sd2k** [reply]
@ivanahuckova yep I did notice that. Unfortunately the ML app isn't really a core app so I'm not sure we can do it in the same way - it feels like we shouldn't bake knowledge of the ML app into Grafana? I could be wrong there though.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
